frontend/src/bundles/common/components/input/input.tsx

```diff
@@ -26,22 +31,48 @@ const Input = <T extends FieldValues>({
     isDisabled = false,
     placeholder = '',
 }: Properties<T>): JSX.Element => {
+    const [isMasked, setIsMasked] = useState(false);
     const { field } = useFormController({ name, control });
 
     const error = errors[name]?.message;
     const hasError = Boolean(error);
+    const isPassword = type === 'password';
+
+    const onMaskPassword = useCallback((): void => {
+        setIsMasked((isMasked) => !isMasked);
+    }, []);
+
+    const placeholderGenerator = (): string => {
+        return isPassword ? placeholder.repeat(6) : placeholder;
+    };
 
     return (
-        <label className="flex h-20 flex-col text-sm">
+        <label className="flex h-20 max-w-[358px] flex-col text-sm">
```

Comment:
```suggestion
        <label className="flex h-20 w-full flex-col text-sm">
```
I think input should take full width because in some case we will need wider inputs

Reply:
Also, don't use px, but use rem